config/taxonomy/base/base.yaml

```diff
@@ -16,4 +16,16 @@ definitions:
   DatasetID:
     type: string
   ProcessingLocation:
-    type: string
\ No newline at end of file
+    type: string
+  Protocol:
+    type: string
+  Dataformat:
+    type: string
+  Interface:
+    type: object
+    properties:
+      protocol:
+        $ref: "#/definitions/Protocol"
+      dataformat:
+        $ref: "#/definitions/Dataformat"
+    additionalProperties: false
```

Comment:
AppInfo needs to be defined in the base.yaml as object similar to Tags

Reply:
I thought AppInfo (role and intent) is not required for a Fybrik application, shouldn't it be an optional layer?

Reply:
AppInfo is required, but intent and role is not required.
See how fybrikApp CRD is defined.